pub_sub/java/http/order-processor/src/main/java/com/service/model/SubscriptionData.java

```diff
@@ -0,0 +1,10 @@
+package com.service.model;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class SubscriptionData<T> {

```

Comment:
Nice.  Unpacks a CloudEvent.  Consider adding comments like C# version.

Reply:
I have provided comments in the form of javadoc. Let me know your thoughts.